migrations/versions/f80726457a72_.py

```diff
@@ -0,0 +1,28 @@
+"""empty message
+
+Revision ID: f80726457a72
+Revises: 0cc35a21afc2
+Create Date: 2018-02-07 16:12:38.279955
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'f80726457a72'
+down_revision = '0cc35a21afc2'
```

Comment:
I believe this revision may need to be modified because of the new version added in the last merge

Reply:
If you change this to `b7852f8aab0a` I think you can get rid of the merge version.